test/StockportContentApiTests/Unit/ContentfulFactories/SpotlightOnBannerContentfulFactoryTest.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Drawing;
+
+namespace StockportContentApiTests.Unit.ContentfulFactories;
+
+public class SpotlightOnBannerContentfulFactoryTest
+{
+    private readonly SpotlightOnBannerContentfulFactory _spotlightOnBannerContentfulFactory;
+
+    public SpotlightOnBannerContentfulFactoryTest() {
+        _spotlightOnBannerContentfulFactory = new SpotlightOnBannerContentfulFactory();
+    }
+
+    [Fact]
+    public void ShouldCreateSpotlightOnListFromAContentfulSpotlightOnList()
+    {
+        List<ContentfulSpotlightOnBanner> entry = new List<ContentfulSpotlightOnBanner>{
+            new ContentfulSpotlightOnBanner {
+                Title = "Spotlight title",
+                Image = new Asset() {
+                    File = new File() {
+                        Url = "ImageUrl"
+                    }
+                },
+                AltText = "AltText",
+                Link = "Link",
+                Sys = new SystemProperties {
+                    PublishedAt = DateTime.MinValue,
+                    UpdatedAt = DateTime.MinValue.AddDays(1)
+                }
+            }
+        };
+
+        var result = _spotlightOnBannerContentfulFactory.ToModel(entry);
+
+        result.Count().Should().Be(1);
+        result.First().Title.Should().Be(entry.First().Title);
+        result.First().Image.Should().Be(entry.First().Image.File.Url);
+        result.First().AltText.Should().Be(entry.First().AltText);
+        result.First().Link.Should().Be(entry.First().Link);
+        result.First().LastUpdated.Should().Be(entry.First().Sys.UpdatedAt);
```

Comment:
This feels like an old way of doing unit tests and they should maybe be Assert.Equal? Not 100% sure